### PR TITLE
Added expo-device package

### PR DESCRIPTION
**Pull request by EvanBacon**

### CODE REVIEW

File: packages/xdl/src/modules/config.ts [hunk 1]
```diff
@@ -309,6 +309,11 @@ const expoUniversalModules = [
     libName: 'expo-gl',
     sdkVersions: '>=29.0.0',
   },
+  {
+    podName: 'EXDeviceInfo',
+    libName: 'expo-device-info',
```
Comment (ide) [inline packages/xdl/src/modules/config.ts]: Just name it `expo-device`

Comment (EvanBacon) [reply packages/xdl/src/modules/config.ts]: CC: @ninjaachibi @vivianzzhu91

Comment (ninjaachibi) [reply packages/xdl/src/modules/config.ts]: > Just name it `expo-device`

got em

**Comment by tsapeta** [comment]
@ninjaachibi, just fyi: when we're adding new modules to xdl config, the corresponding PR on `expo/expo` repo needs to be merged first. Right now some scripts are failing because they use this config and try to find `expo-device` in `packages` folder but it isn't there yet.

**Comment by ninjaachibi** [comment]
Gotcha, thanks @tsapeta!

**Pull request merged**